src/main.ts

```diff
@@ -19,6 +19,12 @@ const argv = yargs
       "Input sourcemap file path. If no output path is provided, the rewritten sourcemap will be written to the same directory with the same name, but with a `.remapped` suffix.",
```

Comment:
This seems like a weird help comment if there isn't custom output file support?

Reply:
I was looking ahead to adding this later :)